pydantic/json_schema.py

```diff
@@ -863,6 +861,7 @@ def _extract_discriminator(
 
     def chain_schema(self, schema: core_schema.ChainSchema) -> JsonSchemaValue:
         # Note: If we wanted to generate a schema for the _serialization_, would want to use the _last_ step:
+        # There are always more than zero steps, since the ChainSchema is validated on the pydantic-core side.
```

Comment:
@dmontagu should we use the last item in the chain if we’re producing the serialization schema